### PR TITLE
feat: support `.eslintrc`

### PR DESCRIPTION
**Pull request by ericclemmons**
Similar to #23.

### CODE REVIEW

**Comment by ericclemmons** [comment]
Cool, I think I got this functional.  If you guys wanna give it a shot, go for it!

**Comment by poeticninja** [comment]
I just tested it out with `example/webpack1`. I ran `npm start`... it installed all of the dependencies. After that I created an `.eslintrc` file with configurations extending `airbnb` as a test. I had to turn off and run `npm start` again for the install to take place again. BUT it worked!! 👍

**Comment by ericclemmons** [comment]
Here's a good test for ya:
> https://www.npmjs.com/package/eslint-config-airbnb

This thing has so many peerDependencies and requirements it's absurd.

**Comment by ericclemmons** [comment]
@insin What do you think about this feature?  Otherwise I'm considering closing it...

File: src/installer.js [hunk 1]
```diff
@@ -119,6 +119,47 @@ module.exports.checkBabel = function checkBabel() {
   this.install(missing);
 };
 
+module.exports.checkESLint = function checkESLint() {
+  try {
+    var options = JSON.parse(
+      fs.readFileSync(
+        path.join(process.cwd(), ".eslintrc"),
```
Comment (michael-ciniawsky) [inline src/installer.js]: There are more possible formats with extensions for e.g `eslintrc.yml, eslintrc.js` and so forth

**Comment by michael-ciniawsky** [comment]
Would this be really needed ?

**Comment by evilebottnawi** [comment]
@michael-ciniawsky imho, seems no

**Comment by michael-ciniawsky** [comment]
👍  || 👎  on my comment above please :)

**Comment by michael-ciniawsky** [comment]
Closing due to inactivity for now, but reopen if still relevant :)

**Pull request closed without merging**